examples/utils/material.py

```diff
@@ -10,6 +10,25 @@
 
 from exabyte_api_client.endpoints.jobs import JobEndpoints
 
+def download_contcar(job_id: str, job_endpoint: JobEndpoints, filename: str):
```

Comment:
This is an example of a non-modular implementation - let's define a `download_file_by_name` function, pls, and pass "CONTCAR" to it on invokation

Reply:
Defined a more general function as requested and changed download_contcar into a partial function of it